packages/kit/package.json

```diff
@@ -60,7 +60,8 @@
 		"build": "rollup -c && node scripts/cp.js src/runtime/components assets/components && npm run types",
 		"dev": "rollup -cw",
 		"lint": "eslint --ignore-path .gitignore --ignore-pattern \"src/packaging/test/**\" \"{src,test}/**/*.{ts,mjs,js,svelte}\" && npm run check-format",
-		"check": "tsc && npm run check:integration && npm run check:prerendering",
+		"check": "tsc",
+		"check:all": "tsc && npm run check:integration && npm run check:prerendering",
```

Comment:
would it make sense for this to also use the `-r` syntax in the same way you updated `test:all`?

Reply:
Oops, I thought I commit that.